ios/GoldWallet.xcodeproj/xcshareddata/xcschemes/GoldWallet.xcscheme

```diff
@@ -50,7 +50,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
@marek-siemieniuk-morawski I know that you changed it recently in the detox PR, so should it be "Release" or "Debug" here?

Reply:
Detox doesn't work with iOS yet. I'll be working later on it so for now it doesn't make any difference for me